1/assets/1.2-finished-code.rs

```diff
@@ -33,9 +35,11 @@ mod incrementer {
     mod tests {
         use super::*;
 
-        #[test]
+        use ink_lang as ink;
+
+        #[ink::test]
```

Comment:
```suggestion
        #[test]
```

Reply:
As described above. While I am not sure it is easier to learn when to use `#[test]` and when to use `#[ink::test]` this workshop should teach people the differences.